jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -95,7 +95,7 @@ blockchain_configuration:
   # if not set, the treasury will not grow
   treasury_parameters:
     # the fix value the treasury will take from the total reward pot of the epoch
-    fixed: 1000
+    fixed: 0
```

Comment:
I don't understand why you needed to change this value?

Reply:
The treasury return does not have a fixed component per epoch.  It is variable based on monetary expansion

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.